scss/_modal.scss

```diff
@@ -48,6 +48,11 @@
   .modal.show & {
     transform: $modal-show-transform;
   }
+
+  // When trying to close, animate focus to scale
+  .modal.modal-static & {
```

Comment:
Only nit is that this could be simplified to `.modal-static &` as long as we're specifying where it goes in the docs. `.modal.show` is chained like that because `.show` is used throughout the project.

Reply:
Unfortunately, the `.modal` (or any class selector) is required for override the above transform definition: 

![image](https://user-images.githubusercontent.com/4065765/67504319-1e118b80-f6c4-11e9-9f07-9e94b642ce0c.png)